index.md

```diff
@@ -17,24 +17,175 @@ redirect_from:
 Thank you!
 ==========
 
-The [DNS flag day 2019](/2019/) was very successful event and the Internet community
-worked together and fixed problems which were causing delays and other
-problems for Internet users worldwide. We would like to thank to all
+The [DNS flag day 2019](/2019/) was very successful event and the Internet
```

Comment:
The Internet community successfully worked together and fixed problems causing delays and other problems for Internet users worldwide on [DNS flag day 2019](/2019/).